next/components/sections/WelcomeSection.tsx

```diff
@@ -49,7 +52,23 @@ export const WelcomeSection = ({ mainMenuItems, homepageHeader }: Props) => {
           />
         </div>
       </div>
-      <HomepageMenu items={mainMenuItems} />
+      <div ref={elementRef} className="w-max h-2 lg:block hidden" />
+      <div className="h-36 w-full relative">
+        {menuState === STICKY_MENU_STATE.VISIBLE && (
```

Comment:
With this approach, we lost animation on sticky header and we are not able to animate opening or closing welcome menu or sticky menu.

I would be better to pass the sate to both components as prop, and decide them wen to open/appear or close/disappear.

You don't have to fix it necessarily in this PR, but the animation on sticky header is important.

You can check the HomePageResults and AnimateHeight components if that animation is helpful :)